src/Symfony/Component/VarDumper/Caster/DOMCaster.php

```diff
@@ -138,16 +138,12 @@ public static function castDocument(\DOMDocument $dom, array $a, Stub $stub, boo
             'doctype' => $dom->doctype,
             'implementation' => $dom->implementation,
             'documentElement' => new CutStub($dom->documentElement),
-            'actualEncoding' => $dom->actualEncoding,
             'encoding' => $dom->encoding,
             'xmlEncoding' => $dom->xmlEncoding,
-            'standalone' => $dom->standalone,
             'xmlStandalone' => $dom->xmlStandalone,
-            'version' => $dom->version,
```

Comment:
Where is this change coming from?

Reply:
It is officially marked as deprecated: https://www.php.net/manual/en/class.domdocument.php#domdocument.props.version